web/styles/settings.css

```diff
@@ -513,14 +513,20 @@ input[type="checkbox"] {
     vertical-align: top;
     height: auto !important;
     width: auto !important;
+    white-space: break-spaces !important;
 
+    overflow-wrap: anywhere;
     background-color: transparent;
     border-radius: 4px;
-    margin-top: 14px;
-    margin-left: 10px;
+    margin-top: 10px;
```

Comment:
If we want this change this should be added only to the notice in accounts section. And if we want this change for all other notices as well, this should be a separate commit.

Reply:
I have made these changes into separate class `account-alert-notification` of accounts section.